### ISSUE TITLE
Is it allowed to use the private framework?

### ISSUE THREAD

**Issue by jerrypainter** [rmaddy/RMPhoneFormat#19]
Hi Rick,

The lib is awesome. But what I am concerning is would apple reject the app because of the using of .dat file?

Thanks so much,

Zixiao

**Comment by taylorhughes** [comment]
No, we've never been rejected for that. Many app versions over the course of years using this library.

**Comment by jerrypainter** [comment]
awesome, thanks!

**Comment by newenglander** [comment]
I've also had no problems using this. @jerrypainter You might want to close this issue.
